homeassistant/components/trafikverket_weatherstation/sensor.py

```diff
@@ -216,21 +215,7 @@ def __init__(
         self._station = sensor_station
         self._weather_api = weather_api
         self._weather: WeatherStationInfo | None = None
-        self._active: bool | None = None
-        self._measure_time: str | None = None
-
-    @property
-    def extra_state_attributes(self) -> dict[str, Any]:
-        """Return the state attributes of Trafikverket Weatherstation."""
-        _additional_attributes: dict[str, Any] = {
-            ATTR_ATTRIBUTION: ATTRIBUTION,
-        }
-        if self._active:
-            _additional_attributes[ATTR_ACTIVE] = self._active
-        if self._measure_time:
-            _additional_attributes[ATTR_MEASURE_TIME] = self._measure_time
-
-        return _additional_attributes
+        self._attr_extra_state_attributes = {ATTR_ATTRIBUTION: ATTRIBUTION}
```

Comment:
Please move the attribution to a class attribute: `_attr_attribution = ATTRIBUTION` and remove the initialisation of `self._attr_extra_state_attributes`

See https://github.com/home-assistant/core/blob/f6c1266af640133fbcc23da98424b7dfaa0e0c75/homeassistant/components/accuweather/sensor.py#L61-L64